composer.json

```diff
@@ -21,7 +21,7 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
+        "php": ">=8.1",
```

Comment:
@adamzimmermann Are you comfortable with dropping 7.4/8.0 support in 1.x?

Reply:
Given that there isn't widespread adoption of this yet, I don't have a concern about breaking things for anyone.

Semantically it probably should technically be a 2.x release, but I'm not stuck on that given the early state of this library. Thoughts?

Reply:
That's fine with me. I just wanted to flag it.

Reply:
Yep, good call as it is a breaking change.

You good with me merging this as is though then?

Reply:
Yes!

Reply:
Oh, I missed your approval earlier. 🙈